pkg/agent/openflow/network_policy.go

```diff
@@ -75,14 +75,17 @@ var (
 	MatchLabelID        = types.NewMatchKey(binding.ProtocolIP, types.LabelIDAddr, "tun_id")
 	MatchTCPFlags       = types.NewMatchKey(binding.ProtocolTCP, types.TCPFlagsAddr, "tcp_flags")
 	MatchTCPv6Flags     = types.NewMatchKey(binding.ProtocolTCPv6, types.TCPFlagsAddr, "tcp_flags")
-	Unsupported         = types.NewMatchKey(binding.ProtocolIP, types.UnSupported, "unknown")
+	// Match value of MatchCTStatePositive will be used as positive CT state match(+).
+	// The value should follow the format "state1,state2".
+	// For example, if the match value is "rpl,trk", it will be translated to ct_state=+rpl+trk.
+	MatchCTStatePositive = types.NewMatchKey(binding.ProtocolIP, types.CTStateAddr, "ct_state")
```

Comment:
I am not super familiar with this code, but IMO it would make more sense to define `MatchCTStateRpl`, use an empty `matchValue` field, and add `ct_state=+trk+rpl` automatically based on that.

Reply:
Done.